seedcase_sprout/core/example_package_properties.py

```diff
@@ -3,23 +3,46 @@
     LicenseProperties,
     PackageProperties,
 )
-
-example_package_properties = PackageProperties(
-    name="example-package",
-    title="Example fake data package",
-    description="Data from a fake data package on something.",
-    contributors=[
-        ContributorProperties(
-            title="Jamie Jones",
-            email="jamie_jones@example.com",
-            roles=["creator"],
-        )
-    ],
-    licenses=[
-        LicenseProperties(
-            name="ODC-BY-1.0",
-            path="https://opendatacommons.org/licenses/by",
-            title="Open Data Commons Attribution License 1.0",
-        )
-    ],
+from seedcase_sprout.core.sprout_checks.check_package_properties import (
+    check_package_properties,
 )
+
+
+def example_package_properties() -> PackageProperties:
+    """Generate an example package properties object.
+
+    Returns:
+        Outputs a correctly formatted, `PackageProperties` object.
+
+    Examples:
+        ```{python}
+        import seedcase_sprout.core as sp
+        sp.example_package_properties()
+        ```
+    """
+    default_properties = PackageProperties.default()
+    update_properties = PackageProperties(
```

Comment:
```suggestion
    updated_properties = PackageProperties(
```
Maybe? If this isn't redundant with Marton's suggestion above (L23).